src/Magento/FunctionalTestingFramework/Util/Validation/NameValidationUtil.php

```diff
@@ -50,4 +68,246 @@ public static function validateName($name, $type)
             throw new XmlException($errorMessage);
         }
     }
+
+    /**
+     * Validates data entity names.
+     *
+     * @param string $str
+     * @param string $filename
+     * @throws Exception
+     * @return void
+     */
+    public function validateDataEntityName($str, $filename = null)
```

Comment:
can this be repurposed as below for all pascal case validations? There won't be a need to separate function for each entity.
```
validateEntityInPascalCase($str, $type, $filename = null)
{
..
$message = "The ($type) {$str} should be PascalCase with an uppercase first letter.";
..
}
```

Reply:
fixed